README.md

```diff
@@ -1,29 +1,29 @@
-# ZeppelinOS library
+# ZeppelinOS library _(zos-lib)_
+
 [![NPM Package](https://img.shields.io/npm/v/zos-lib.svg?style=flat-square)](https://www.npmjs.org/package/zos-lib)
+
 [![Build Status](https://travis-ci.org/zeppelinos/zos-lib.svg?branch=master)](https://travis-ci.org/zeppelinos/zos-lib)
+
```

Comment:
This spaces leaves each badge in its own line, which looks weird and takes lots of space.